### ISSUE TITLE
Why cant i sell more stacks?

### ISSUE THREAD

**Issue by Kingwersens** [Shopkeepers/Shopkeepers#619]
## Preliminaries:

* **Shopkeepers version:** `<Insert Shopkeepers version>`
* **Spigot version:** `<Insert output of /version command>`
---

* I have checked that my issue/question does not get answered by:
  * The [documentation](https://github.com/Shopkeepers/Shopkeepers-Wiki/wiki).
  * The [FAQ](https://github.com/Shopkeepers/Shopkeepers-Wiki/wiki/Frequently-Asked-Questions).
  * The [Known Issues](https://github.com/Shopkeepers/Shopkeepers-Wiki/wiki/Known-Issues).
* I have checked all open and closed [issues](https://github.com/Shopkeepers/Shopkeepers/issues), but none seems to fit my issue/question.

## Reproduction on a fresh and up-to-date Spigot server:

I was **`<able / not able>`** to reproduce my issue on a freshly setup and up-to-date Spigot server (currently **`<Insert output of /version command on up-to-date Spigot server>`**) with the latest version of Shopkeepers (currently **`<Insert latest tested Shopkeepers version>`**), with no other plugins and with no kinds of other server or client mods.

## The issue:

Provide a detailed description of your problem and add as much information as possible which could help reproducing the issue.

Include:
* Description of the issue.
* Step-by-step instructions on how to reproduce the issue.
* Observed behavior.
* Expected behavior.
* If items with special attributes are involved: Include instructions on how to create those items.
* Server logs (from server startup until issue). Consider enabling debug mode in the config for additional console output.
* If modified from default: Plugin configurations.
* .. Anything else that might be useful to reproduce and identify the issue.

**Comment by blablubbabc** [comment]
Shopkeepers uses minecraft's trading interface which only supports trades of one item stack for up to two input item stacks.
What you can do is shift click to trade multiple times (for the same conditions), like it is possible in vanilla minecraft. But you can't define prices for more than one item stack.

This might principally be possible with fundamental changes to:
* How trades are setup: Players would need to be able to specify item stack sizes beyond the minecraft limits inside the editor, and possibly (if this were to be supported as well) multiple item stacks of different types.
* How trades are represented inside the trading interface: A trading player would need to be able to determine which items and how many of them he receives. A possible solution would be to specify this inside the result item's lore text (with the drawback that, unlike it is known from trading in vanilla minecraft, the lore of the item displayed inside the result slot doesn't exactly match the received item anymore).
* And processed: The plugin would have to deal with handing out multiple result items and with how to inform the player about possible issues, like not having enough space in inventory and the like.

So basically this would require changes to all aspects of how the trading with this plugin currently works, making it a different plugin in that aspect and diverging from the vanilla trading logic even fruther. Currently I considered this to be out of scope for this plugin.

**Comment by OscarVanL** [comment]
I would love this feature, but it does seem unlikely given what you just said.
